Lombiq.Tests.UI/Extensions/FormWebDriverExtensions.cs

```diff
@@ -10,6 +10,8 @@ public static IWebElement TryFillElement(this IWebDriver driver, IWebElement ele
     {
         element.ClearWithLogging();
 
+        if (string.IsNullOrEmpty(value)) return element;
```

Comment:
It's valid to empty out an input element, i.e. fill it with an empty string. Why this shortcut?

Reply:
The method has already cleared out the input by this point using the `element.ClearWithLogging();` in the line above. So if the value is `null` or empty, there is nothing further to do. Also, if the value was actually `null` we'd get an NRE. on the next line.